script.sh

```diff
@@ -7,13 +7,15 @@ TEMP_PATH="$(mktemp -d)"
 PATH="${TEMP_PATH}:$PATH"
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
+STATICCHECK_VERSION="2024.1.1"
 echo '::group:: Installing staticcheck ... https://staticcheck.io'
-curl -sfL  "https://github.com/dominikh/go-tools/releases/latest/download/staticcheck_linux_amd64.tar.gz" | tar -xvz -C "${TEMP_PATH}" --strip-components=1
+curl -sfL  "https://github.com/dominikh/go-tools/releases/download/${STATICCHECK_VERSION}/staticcheck_linux_amd64.tar.gz" | tar -xvz -C "${TEMP_PATH}" --strip-components=1
+staticcheck --version
 echo '::endgroup::'
 
 
 echo '::group:: Running staticcheck with reviewdog 🐶 ...'
-staticcheck ${INPUT_STATICCHECK_FLAGS} -f=json ${INPUT_TARGET:-.} \
+staticcheck "${INPUT_STATICCHECK_FLAGS}" -f=json "${INPUT_TARGET:-.}" \
```

Comment:
Please unquote INPUT_STATICCHECK_FLAGS and INPUT_TARGET.
Space in these env vars should be treated as-is. 

You can ignore shellcheck warning.